website/docs/reference/opspec/op.yml/predicate.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Predicate
+---
+
+A predicate is a condition that evaluates to a boolean value, used for conditional logic. The predicate is an object with a single key defining the type of condition.
+
+## `eq`
+
+An [array](../types/array). When all items in the array are equivalent, the predicate is true.
```

Comment:
@chrisdostert Is type coercion applied during `eq`/`ne` predicate evaluation? (related to https://github.com/opctl/opctl/issues/909)

Reply:
currently yes